.github/workflows/release.yml

```diff
@@ -180,11 +102,6 @@ jobs:
         run: |-
           git config user.name "AWS PDK Automation"
           git config user.email "aws-pdk+automation@amazon.com"
-      - name: Upload docs to Github
-        run: zip -r docs.zip dist/public/docs/dist/docs/* && gh release upload $(cat dist/packages/nx-monorepo/dist/releasetag.txt) -R $GITHUB_REPOSITORY docs.zip && rm docs.zip
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GITHUB_REPOSITORY: ${{ github.repository }}
       - name: Prepare Commit
```

Comment:
Need to double check this, should it be removed?

Reply:
This was previously storing the whole docs as a zip so users could view docs at a 'point in time'. I don't think this makes sense any more given we no longer release as a whole. The docs are also now contained within the package so it should be fine to remove.